doc/api/child_process.md

```diff
@@ -334,18 +340,8 @@ getVersion();
 function. Any input containing shell metacharacters may be used to trigger
 arbitrary command execution.**
 
-If the `signal` option is enabled, calling `.abort()` on the corresponding
-`AbortController` is similar to calling `.kill()` on the child process except
-the error passed to the callback will be an `AbortError`:
-
-```js
-const controller = new AbortController();
-const { signal } = controller;
-const child = execFile('node', ['--version'], { signal }, (error) => {
-  console.log(error); // an AbortError
-});
-controller.abort();
-```
+The `signal` option works exactly the same way it does in
```

Comment:
I actually think examples in APIs are better than not - since users often don't know all the APIs they're working with - sending them to another API (spawn) when they're trying to use execFile isn't better than an example using execFile IMO.

The fact internally execFile delegates to spawn to save code internally is an implementation detail